server/routes/incidents.test.js

```diff
@@ -94,68 +58,4 @@ describe('GET /your-report', () => {
         expect(res.text).toContain('Use of force report')
       })
   })
-
-  it('should fail to render page when no report', () => {
```

Comment:
Do you not want to keep this test in for `/your-report`? This is now only testing the happy path for this endpoint now isn't it?

Reply:
the logic has been pushed down to the service - so we don't explicitly need to test it at this level, its just an unexpected error